### ISSUE TITLE
Error:   Class 'GuzzleHttp\Psr7\Utils' not found - Upgrading Laravel 5.1 to Laravel 5.2 

### ISSUE THREAD

**Issue by krixnaas** [guzzle/guzzle#2984]
[Symfony\Component\Debug\Exception\FatalThrowableError]
  Class 'GuzzleHttp\Psr7\Utils' not found

..composer.json
"guzzlehttp/guzzle": "^6.5",
"guzzlehttp/psr7": "^1.8",
"guzzlehttp/promises": "^1.3.1"

**Comment by GrahamCampbell** [comment]
Guzzle 6 is EOL, though I appreciate Laravel 5.2 does not support 7 (though you could try it).

Try running `composer update` with the following, for Guzzle 6:

```json
"guzzlehttp/guzzle": "^6.5.5",
"guzzlehttp/psr7": "^1.8.3",
"guzzlehttp/promises": "^1.5.1"
```

You should also check to see if Guzzle is installed globally, and run a composer global update, if it is.

**Comment by GrahamCampbell** [comment]
TBH, I'd not recommend upgrading from Laravel 5.1 to 5.2. 5.1 got bug and security fixes for much longer than 5.1. I'd try and go straight from 5.1 to 5.5 if I were you, then straight from 5.5 to 6 as soon as possible after.
